apps/src/code-studio/components/libraries/LibraryCreationDialog.jsx

```diff
@@ -100,11 +100,17 @@ class LibraryCreationDialog extends React.Component {
         this.setState({publishState: PublishState.NO_FUNCTIONS});
         return;
       }
+      let librarySource = response.source;
+      response.libraries &&
```

Comment:
nit: I think this would be clearer as `if (response.libraries) {...}`

Reply:
👍